routes/golfer_profile.go

```diff
@@ -28,7 +28,8 @@ func golferGET(w http.ResponseWriter, r *http.Request) {
 	}
 
 	db := session.Database(r)
-	golfer := session.GolferInfo(r).Golfer
+	golferInfo := session.GolferInfo(r)
+	golfer := golferInfo.Golfer
```

Comment:
So that's not quite what I meant, because the golfer info struct is a superset it should just be possible to only have that reference. So:

```go
	db := session.Database(r)
	golfer := session.GolferInfo(r)
```
Nothing else.